examples/simple-proposals.yaml

```diff
@@ -179,3 +180,83 @@
     '@type': ProposedIntent
     publishedIn: alice:8e5fe80d-a769-4bd5-89e5-2136d33eab9f # the proposal
     publishes: alice:02b39a30-3e04-4305-9656-7f261aa63c84 # the intent (this is the same intent alice originally published)
+
+  # Price list
+
+  - '@id': farm:da7bfa38-a1b2-4f6f-a896-b9bf1ec8807a
+    '@type': Proposal
+    hasBeginning: 2019-03-01T00:00:00-5:00
+    hasEnd: 2019-03-31T00:00:00-5:00
+    eligibleLocation: map:566a2c01-96f8-4b3c-adcf-f197eced3092 # Madison, WI
+
+  - '@id': farm:c7897c39-7f05-4a5d-a487-80e130a24345
+    '@type': Intent
+    name: Carrots case 25#
+    action: transfer
+    provider: https://farm.example/
+    resourceClassifiedAs: wd:Q81 # carrot
+    resourceSpecifiedAs: farm:2402b57c-1c45-423a-acb2-f50961c97d65 # 25 pound case organic carrots seconds
+    exchangeRateQuantity:
```

Comment:
Intent itself shouldn't have any notion of 'exchange rate' since same intent can get referenced from any number of proposals. I guess you could try moving it into Proposed Intent which stays specific to single Proposal.

Reply:
I think maybe I named this badly.  This is only one side, it is not a "rate" in that sense.  This is for the "price list" use case, like 25# case carrots.  In the other intent would be $32.  So 25# case carrots is re-usable, and would be there as long as carrots are in stock for the season (every week a new price list), and also re-usable year after year.  The $ side might vary somewhat over the season, but not necessarily.

It is also a good portion of retail e-commerce, which list unit prices, and you can buy however many you want, within some large limit.  Also might be most of retail, although those usually aren't published outwardly facing on a computer.  But still it is the same data.

I don't totally mind putting it in the ProposedIntent, but think it might be more confusing to apps, and don't see the need.

Maybe "unitQuantity" - or I think you had "scalableQuantity" or similar....?

Reply:
What if for the same intent of providing *undefined* quantity of 25# boxes of carrot, farm would want to have two different proposals (one for bulk amounts)?
I think this kind of information stays related to proposal.
I guess intent could have information about 'atomic quantity', commitments satisfying it would need to always have quantity multiplied by positive natural number.

Reply:
I think an intent for undefined bulk amounts is a different intent, and different proposal.  And even a different resource.

Reply:
>I don't totally mind putting it in the ProposedIntent, but think it might be more confusing to apps, and don't see the need.

Actually, I think it needs to stay in Intent, otherwise the Intent is not re-usable - or it is re-usable but not usefully.  It also fits from a modeling perspective in the same entity that has the resource class/spec.  Just needs to be distinguished from other quantity(s).

Reply:
Changed name to unitQuantity.